sim/CMakeLists.txt

```diff
@@ -36,8 +52,11 @@ endif(NOT WIN32)
 
 ## Test Dependencies
 # Google test
+set(GOOGLETEST_VERSION 1.8.1)
```

Comment:
Googletest version is tracked by git subtree, so i don't think listing the version here does anything./

Reply:
I added this because every time I run configure, I get a warning:
```
CMake Warning at lib/googletest/googletest/CMakeLists.txt:54 (project):
  VERSION keyword not followed by a value or was followed by a value that
  expanded to nothing.
```

Reply:
Oops, looks like i misconfigured this according to [this googletest issue](https://github.com/google/googletest/issues/2950). Should be just  `add_subdirectory("${LIB_DIR}/googletest", EXCLUDE_FROM_ALL)`

Reply:
Fixed in 3016255